statsd/statsd_benchmark_test.go

```diff
@@ -2,16 +2,22 @@ package statsd_test
 
 import (
 	"fmt"
+	"io"
 	"log"
 	"net"
 	"os"
+	"runtime"
+	"strconv"
+	"strings"
 	"sync/atomic"
 	"testing"
 
 	"github.com/DataDog/datadog-go/statsd"
 )
 
-func setupUDSClientServer(b *testing.B) (*statsd.Client, net.Listener) {
+var goMinorVersion, _ = strconv.Atoi(strings.Split(runtime.Version(), ".")[1])
```

Comment:
nit: I don't think this variable is actually used. I guess you wanted to use it to report the additional drop metric only on go1.13 and up?

To do so you could put this logic in a different test file behind the `go1.13` build tag, see https://golang.org/pkg/go/build/#hdr-Build_Constraints

Reply:
You read my mistakes like an open book ;)